tests/rados/mute_alerts.py

```diff
@@ -0,0 +1,351 @@
+        log.error("could not generate the error")
+        return False
+
+    duration = kwargs.get("duration")
+    sticky = kwargs.get("sticky")
+
+    if not mute_health_alert(alert=alert, node=node, duration=duration, sticky=sticky):
+        log.error(f"could not mute the alert : {alert}")
+        return False
+
+    return True
+
+
+def get_alerts(node: CephAdmin) -> dict:
+    """
+    Fetches all the current health alerts codes that are generated on the ceph cluster
+    Args:
+            node: node on which command should be executed
+
+    Returns: list of the alerts present
+            alert dictionary :
+            { "active_alerts" : ['CEPHADM_REFRESH_FAILED', 'OSDMAP_FLAGS'],
+             "muted_alerts" : ['MON_DISK_BIG'] }
+    """
+    cmd = "ceph health detail"
+    all_alerts = {}
+    out, err = node.shell([cmd])
+    regex_active = r"\s*[^(MUTED)]\s+\[\w{3}\]\s([\w_]*):"
+    regex_muted = r"\s*[(MUTED)]\s+\[\w{3}\]\s([\w_]*):"
+    all_alerts["active_alerts"] = re.findall(regex_active, out)
+    all_alerts["muted_alerts"] = re.findall(regex_muted, out)
+    return all_alerts
+
+
+def mute_health_alert(
+    alert: str, node: CephAdmin, duration: str = None, sticky: bool = False
+) -> bool:
+    """
+    Mutes the health alert generated on the cluster
+    Args:
+        alert: Name of the alert to be muted
+        node: node on which command should be executed
+        duration: duration for which the alert should be muted.
+                Allowed Values: None -> mutes the specified alert indefinitely until the same alert is raised again
+                                5m, 1h -> mutes the specified alert for specified duration
+        sticky: makes use of the "--sticky" param to mute specified alert indefinitely
+
+    Returns: True -> pass, False -> failure
+
+    """
+    all_alerts = get_alerts(node)
+    if alert not in all_alerts["active_alerts"] + all_alerts["muted_alerts"]:
+        log.info(f"the alert: {alert} not generated on the cluster, Cannot mute")
+        return True
+    if alert in all_alerts["muted_alerts"]:
+        log.info(f"the alert: {alert} is already muted")
+        return True
+
+    # Muting the given alert along with specified duration
+    cmd = f"ceph health mute {alert}"
+    if duration:
+        cmd += f" {duration}"
+    if sticky:
+        cmd += " --sticky"
+    node.shell([cmd])
+
+    # Sleeping for 2 sec for the alert to be logged
+    time.sleep(2)
+    all_alerts = get_alerts(node)
+    log.info(
+        f"Muted the alert : {alert}. All the muted alerts : {all_alerts['muted_alerts']}"
+    )
+    return True if alert in all_alerts["muted_alerts"] else False
+
+
+def unmute_health_alert(alert: str, node: CephAdmin) -> bool:
+    """
+    Un-Mutes the health alert on the cluster
+    Args:
+        alert: Name of the alert to be muted
+        node: node on which command should be executed
+
+    Returns: True -> pass, 1 -> failure
```

Comment:
I believe it should be `False --> Failure`.. the rest looks good to be me.

Reply:
done